### PR TITLE
Add Audit log stream API (GEA-13590)

### PR DESCRIPTION
**Pull request by kenany**

### CODE REVIEW

File: packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/audit/LogStreamRequest.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package cloud.pangeacyber.pangea.audit;
+
+import cloud.pangeacyber.pangea.BaseRequest;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public final class LogStreamRequest extends BaseRequest {
+
+	@JsonProperty("logs")
```
Comment (pangea-andrest) [inline packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/audit/LogStreamRequest.java:9]: Do they have to be in a `logs` field? Will the backend will parse them as multiple events?
If they have to be inside `logs` field we could do a class that has at least this field.

Comment (kenany) [reply packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/audit/LogStreamRequest.java:9]: It's how one provider does it, yes, but future providers might not.

**Review by pangea-andrest — APPROVED** [review]

**Review by pangea-andrest — APPROVED** [review]

File: packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/BaseClient.java [hunk 2]
```diff
@@ -160,7 +163,8 @@ public abstract class BaseClient {
 	private static final ObjectMapper objectMapper = JsonMapper
 		.builder()
 		.findAndAddModules()
-		.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+		.defaultTimeZone(TimeZone.getTimeZone(ZoneOffset.UTC))
+		.withConfigOverride(Instant.class, cfg -> cfg.setFormat(JsonFormat.Value.forPattern("uuuu-MM-dd'T'HH:mm:ss.SSSSX")))
```
Comment (pangea-andrest) [inline packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/BaseClient.java:167]: Is it okey to have only 4 seconds' decimals? I think backend expect to have 6 (microseconds) at least in standard schema.

Comment (kenany) [reply packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/BaseClient.java]: My bad, thought it was 4 digits. I couldn't find this documented anywhere on the website.

**Review by pangea-andrest — APPROVED** [review]
Nit: Commit comment should not be nanoseconds but microseconds

**Pull request merged**